src/apple.rs

```diff
@@ -0,0 +1,25 @@
+use bevy::prelude::*;
+use bevy::core::FixedTimestep;
+
+use crate::board::Board;
+
+pub struct Plugin;
+
+impl bevy::prelude::Plugin for Plugin {
+    fn build(&self, app: &mut App) {
+        app.add_system_set(
+                SystemSet::new()
+                    .with_run_criteria(FixedTimestep::step(5.0))
+                    .with_system(add_apple)
+            );
+    }
+}
+
+#[derive(Component)]
+pub struct Apple;
+
+fn add_apple(mut commands: Commands, mut board: ResMut<Board>) {
```

Comment:
This is the major design decision. How much game state/knowledge should be encapsulated in Board vs. the ECS?